gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
```

Comment:
It is not sufficient to just change the URL. You also need to update the wrapper script itself. See https://gradle.org/install/#with-the-gradle-wrapper
When you execute that, both the properties and the wrapper should be updated.

Reply:
What script (gradlew ?) ant which point should be updated?

Reply:
Ok, will do it.